neural_modelling/src/neuron/plasticity/weight_change/post_events_with_weight_change.h

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2017 The University of Manchester
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//! \file
+//! \brief Post-synaptic events
+#ifndef _POST_EVENTS_H_
```

Comment:
Used in three different h files!

Reply:
Probably not a big problem.  Arguably, they can't be used at the same time... could be fixed if that is somehow needed later!